src/ingest_validation_tools/enums.py

```diff
@@ -42,17 +42,16 @@
         'MxIF',
         'MALDI-IMS',
         # 'MS (shotgun lipidomics)',
+        'Multiplex Ion Beam Imaging',
         # 'NanoDESI',
         # 'NanoPOTS',
         'PAS microscopy',
-        # 'scATACseq',
```

Comment:
@mccalluc I can add a commit to remove scATACseq from the schema.  Unfortunately the name of the whole schema is 'scatacseq'. AFAIK there is no requirement that the schema name match an assay name, so we could have scatacseq-v1 with no scATACseq.  Would that be problematic from your point of view?  Should the names of the files be changed to, say, snatacseq-v1 as well?  The alternative would be to create the assay on the search-api side, but it seems wrong to create an assay with a full set of characteristics when there may never be data of that type.

Reply:
> we could have scatacseq-v1 with no scATACseq. Would that be problematic from your point of view?

I don't think it would be problem technically: The only places that end-users see those strings are URLs under https://hubmapconsortium.github.io/ingest-validation-tools/ 

> Should the names of the files be changed to, say, snatacseq-v1 as well?

That would be fine with me, too: for the end user, it only means that the the ghpages URLs change.

I'd suggest getting input from Nils and Chris before changing.

Reply:
Sounds good. 
Let's discuss further with Nils.

Reply:
I have added commits to remove scATACseq from the table schemas (without changing the filenames).